src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -60,7 +63,83 @@
 ]
 
 
-def handle_exception(ex):  # pylint: disable=too-many-return-statements
+class AzCLIErrorType(Enum):
+    """ AzureCLI error types """
+
+    # userfaults
+    CommandNotFoundError = 'CommandNotFoundError'
+    ArgumentParseError = 'ArgumentParseError'
+    ValidationError = 'ValidationError'
+    ManualInterrupt = 'ManualInterrupt'
```

Comment:
KeyboardInterrupt is clear enough, what's your thoughts?

Reply:
Just for consistency, KeyboardInterrupt is good for me. Will use `KeyboardInterrupt ` to replace it.